### DISCUSSION TITLE
Carlinkit 2.0 CPC200-U2W Unbricking

### DISCUSSION THREAD

**Discussion by Ucippa** [ludwig-v/wireless-carplay-dongle-reverse-engineering#159] [General]
_Originally posted by @evilxk in https://github.com/ludwig-v/wireless-carplay-dongle-reverse-engineering/discussions/72#discussioncomment-1660590_

**Comment by Ucippa** [comment]
@ludwig-v @evilxk Hi to evrerybody, i've found this old post and i've used your dump file to resume my old carlinkit 2.0 CPC200-U2W that was bricked after a FW upgrade. I've used CH341a programmer without error, but when i connect the dongle led green and red blinking alternately, first slowly, then quickly and then they go off, and it doesn't happen anymore...the dongle seems dead. do you have any suggestion?

**Reply by andrey-kk** [reply]
Hey, I have exactly the same symptoms. Already tried 3 different firmwares.

**Reply by Quikeramos1** [reply]
I have the same situation, but if I wait for less than a minute, the led turn red again and it works, but I receive a activation error message.

**Comment by ludwig-v** [comment]
Try different backups, some are working for some people, some others are not ;)
https://github.com/ludwig-v/wireless-carplay-dongle-reverse-engineering/tree/master/Flash_Dump

**Reply by Ucippa** [reply]
Thanks for your suggestion. i'm trying different backups, but now i receive a "verification error" during IC programming (...also with old bin file!!). Could be possible that i've damaged the IC ?

**Comment by omercnet** [comment]
@Ucippa you need to erase before you write
AsProgrammer as has a combined erase+write+verify button

**Reply by Ucippa** [reply]
Hi, I had to buy a new 25L12835F because the original one was definitely broken. I would then have to rewrite the new IC with the correct firmware. Which .bin dump should I use considering I have an old CPC200-U2W? For writing I have a CH341 programmer.

Thank you !!
